subsys/bluetooth/controller/ll_sw/nordic/lll/lll_central_iso.c

```diff
@@ -155,6 +155,7 @@ static int prepare_cb(struct lll_prepare_param *p)
 
 	/* Get reference to ACL context */
 	conn_lll = ull_conn_lll_get(cis_lll->acl_handle);
+	LL_ASSERT(conn_lll);
```

Comment:
```suggestion
	LL_ASSERT(conn_lll != NULL);
```

Applies many places

Reply:
It's been in our coding guidelines for quite a while actually: Rule 85 of https://docs.zephyrproject.org/latest/contribute/coding_guidelines/index.html: https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_14_04.c

Reply:
I added a commit and a revert commit as there is code size increase in use of explicit comparison.

The assertion check is correct to have, but the explicit checks seems an overhead due to coding style (both Bluetooth subsystem and few kernel code break this style anyway).

I have come to realization that these invalid checks like LL_ASSERT; and explicit comparisons total ~15 KB of code size on Cortex-M CPUs in the Controller hci_uart builds!

Reference exercise: #90599

Reply:
That's surprising :o Do you know why changing from e.g. `conn_lll)` to `conn_lll != NULL` increase the code size?

Reply:
Apparently, on investigation, the assert string includes the conditional as string! ... i will put back the MISRA coding style.